### PR TITLE
hwdef: Create f405-MatekAirspeed

### PR DESCRIPTION
**Pull request by hendjoshsr71**
This creates a hwdef specifically for the f405-Matek DLVR sensor. Previously, they shared the same hwdef. Note the airspeed sensor shares a board ID with the GPS.  @MATEKSYS 

The benefit of this is we can turn off by default some of the enabled functionality such as the compass and battmonitor so we do not flood the CAN network with traffic for sensors that aren't attached. A user still can go into the DroneCAN GUI Tool to change use a BATT_Monitor when desired.

Also added are a few defines so that disabling baro and compass can be done from the hwdef file.
Cleaned up some of the duplication in the GPS's defaults.parm

Note to set the default rangefinder port correctly #19905 is needed.

image

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/f405-MatekAirspeed/defaults.parm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# setup for Neopixel
+OUT7_FUNCTION 120
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/f405-MatekAirspeed/defaults.parm:2]: the real board doesn't have a neopixel

File: libraries/AP_HAL_ChibiOS/hwdef/f405-MatekGPS/hwdef.dat [hunk 2]
```diff
@@ -194,6 +198,9 @@ define AP_PERIPH_RANGEFINDER_PORT_DEFAULT -1
 
 define HAL_PERIPH_ENABLE_ADSB
 
+# disable ADSB by default
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/f405-MatekGPS/hwdef.dat:201]: lets remove ADSB from this build

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/f405-MatekAirspeed/defaults.parm (hunk 1 above)
Comment (hendjoshsr71) [reply libraries/AP_HAL_ChibiOS/hwdef/f405-MatekAirspeed/defaults.parm]: Removed

File: libraries/AP_HAL_ChibiOS/hwdef/f405-MatekGPS/hwdef.dat (hunk 2 above)
Comment (hendjoshsr71) [reply libraries/AP_HAL_ChibiOS/hwdef/f405-MatekGPS/hwdef.dat]: Removed

**Review by tridge — APPROVED** [review]

**Pull request merged**